src/sentry/static/sentry/app/components/events/rrwebReplayer.jsx

```diff
@@ -0,0 +1,34 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import 'rrweb-player/dist/style.css';
+import rrwebPlayer from 'rrweb-player';
+
+export default class RRWebReplayer extends Component {
+  static propTypes = {
+    url: PropTypes.string.isRequired,
+  };
+
+  constructor(...params) {
+    super(...params);
+    this.ref = React.createRef();
+  }
```

Comment:
I left this comment elsewhere, but `ref` can just be an [instance field](https://2ality.com/2017/07/class-fields.html#instance-fields).

```tsx
export default class RRWebReplayer extends Component {
   ref = React.createRef();
   ...
}